Tests/test_PDB_MMCIF2Dict.py

```diff
@@ -26,37 +26,305 @@
+        # Turn black code style off
+        # fmt: off
+        self.assertEqual(mmcif["_entity_poly_seq.mon_id"], ["MSE", "ASP", "ILE", "ARG",
```

Comment:
Maybe use a more black-like style here, e.g. line break after the ``[`` and put  ``"MSE", "ASP", "ILE", "ARG", "GLN", "GLY", "PRO", "LYS", ...`` on the second line (only four more spaces indented than the self line). Maybe 10 resides per line would fit comfortably?